lib/ContentHelpers.php

```diff
@@ -149,10 +149,21 @@ public static function extractFlexibleContent(Entry $entry, $locale, $children =
         foreach ($entry->flexibleContent->all() as $block) {
             switch ($block->type->handle) {
                 case 'contentArea':
+
+                    // Scale down any in-body images
+                    $contentWithResizedImages = preg_replace_callback(
+                        '/<img src="(https:\/\/www.tnlcommunityfund.org.uk\/media\/.+?)"/i',
```

Comment:
Noting here that this won't match if something comes before `src`, guess we are relying on craft having a standard way of outputting with `src` as the first attribute which is probably OK. Prefer this to something more generic but more prone to false positives.